shell/indicators/StorageIndicator.qml

```diff
@@ -32,68 +32,35 @@ import Hawaii.Hardware 1.0 as Hardware
 import ".."
 
 Indicator {
-    name: "storage"
+    name: qsTr("Storage")
```

Comment:
the idea was to use it when loading an indicator from a plugin, but it's an early concept that was never developed. it can be renamed to title with later commits.